v2.0/admin-ui-runtime-dashboard.md

```diff
@@ -51,9 +51,6 @@ Metric | Description
 --------|----
 User CPU Time | Total CPU seconds per second used by the CockroachDB process across all nodes.
 Sys CPU Time | Total CPU seconds per second used by the system calls made by CockroachDB across all nodes.
-GC Pause Time | Time required by the Garbage Collection process of Go.
```

Comment:
The GC Pause Time still shows up in the screenshot. I know we haven't versioning the images dir yet, but for now, can you just add a 2.0-specific image and use it here?

Reply:
Done!